### PR TITLE
Update node driver known issues

### PR DESCRIPTION
**Pull request by futuretea**
Update node driver known issues
- [v1.1] update the issue status of  https://github.com/harvester/harvester/issues/3272
- [v1.2] add the known issue

**Related issues**
https://github.com/harvester/harvester/issues/3272

### CODE REVIEW

File: docs/rancher/node/node-driver.md [hunk 1]
```diff
@@ -31,6 +31,61 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
...
+| Resolved | 2023-05-08   |
+
+**Workaround**: You can temporarily change the Harvester node driver version to [v0.6.3](https://github.com/harvester/docker-machine-driver-harvester/releases/tag/v0.6.3) from the Rancher UI.
+1. Go to the Rancher UI and click `Cluster Management` > `Drivers` > `Node Drivers`. In the `Node Drivers` list, find ` Harvester` and then click `⋮`  > `View in API`.
+2. Click `Edit`.
+3. Uncheck the `builtin` checkbox.
+4. Change the `*url` to `https://releases.rancher.com/harvester-node-driver/v0.6.3/docker-machine-driver-harvester-amd64.tar.gz`.
+5. Change the `checksum` to `159516f8f438e9b1726418ec8608625384aba1857bc89dff4a6ff16b31357c28`.
+6. Click `Show Request` > `Send Request`.
+7. Click `Reload` util the value of `status.appliedChecksum` and `status.appliedURL` change to the value we set.
```
Comment (vickyhella) [inline docs/rancher/node/node-driver.md:52]:
```suggestion
7. Click `Reload` until the value of `status.appliedChecksum` and `status.appliedURL` change to the value we set.
```

File: docs/rancher/node/node-driver.md [hunk 2]
```diff
@@ -31,6 +31,61 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
 ### Support Matrix
 Refer to [Rancher Downstream Cluster Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-9)
 
+## Known Issues
+
+| Summary                                                                                                                                                                                   | Status    | Last Updated |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
+| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Resolved | 2023-05-08   |
+
+### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster.
```
Comment (vickyhella) [inline docs/rancher/node/node-driver.md:40]:
```suggestion
### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster
```

**Review by vickyhella — APPROVED** [review]
Just some nits, the rest LGTM! 
Would you please also apply these suggestions to the v1.1 doc? Thanks ^^

File: versioned_docs/version-v1.1/rancher/node/node-driver.md [hunk 3]
```diff
@@ -35,12 +35,12 @@ Refer to [Rancher Downstream Cluster Support Matrix](https://www.suse.com/suse-r
...
 |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
-| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Mitigated | 2023-03-15   |
+| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Resolved | 2023-05-08   |
 
-### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster.
+### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster
 | Status    | Last updated |
 |-----------|--------------|
-| Mitigated | 2023-03-15   |
+| Resolved | 2023-05-08   |
```
Comment (guangbochen) [inline versioned_docs/version-v1.1/rancher/node/node-driver.md:43]:
```suggestion
| Resolved(Rancher >=v2.7.2) | 2023-05-08   |
```

File: docs/rancher/node/node-driver.md [hunk 4]
```diff
@@ -31,6 +31,61 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
...
+## Known Issues
+
+| Summary                                                                                                                                                                                   | Status    | Last Updated |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
+| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Resolved | 2023-05-08   |
+
+### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster
+| Status    | Last updated |
+|-----------|--------------|
+| Resolved | 2023-05-08   |
```
Comment (guangbochen) [inline docs/rancher/node/node-driver.md:43]:
```suggestion
| Resolved(Rancher >= v2.7.2)  | 2023-05-08   |
```

**Review by guangbochen — APPROVED** [review]

**Pull request merged**